app/src/main/java/subins2000/manglish/MainActivity.java

```diff
@@ -1,75 +1,96 @@
 /**
  * Manglish, malayalam to manglish converter
  * Copyright (C) 2019 Subin Siby
- *
+ * <p>
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, version 3 of the License
- *
+ * <p>
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- *
+ * <p>
```

Comment:
Why this `<p>` though ?

Reply:
Hmm... I think it added automatically :)